app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -51,6 +51,8 @@
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="28sp"
+      android:textAlignment="viewStart"
+      app:layout_constraintStart_toEndOf="@+id/profile_edit_image"
```

Comment:
Why these changes?

Reply:
Sorry I think the previous issue code changes might be pushed ... now it is removed @anandwana001